analyzers/src/SonarAnalyzer.CSharp/Rules/ClassShouldNotBeEmpty.cs

```diff
@@ -18,6 +18,8 @@
  * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  */
 
+using System.Xml.Linq;
```

Comment:
Not needed.

Reply:
I hate it when Visual Studio adds `usings` without telling you. It has been removed now.